python/tank/authentication/shotgun_authenticator.py

```diff
@@ -188,7 +188,7 @@ def _create_session_user(
         return user.ShotgunUser(impl)
 
     def create_session_user(
-        self, login, session_token=None, password=None, host=None, http_proxy=None
+        self, login, session_token=None, password=None, host=None, http_proxy=None, session_metadata=None
```

Comment:
At this point, I was thinking of simply removing this method ans simply rename _create_session_user to remove the _ prefix...